x-pack/plugins/apm/server/lib/anomaly_detection/create_anomaly_detection_jobs.ts

```diff
@@ -136,12 +144,14 @@ async function createAnomalyDetectionJob({
       ],
     });
 
-    await waitForIndexStatus({
-      client: esClient,
-      index: '.ml-*',
-      timeout: DEFAULT_TIMEOUT,
-      status: 'yellow',
-    })();
+    if (!serverless) {
+      await waitForIndexStatus({
+        client: esClient,
+        index: '.ml-*',
+        timeout: DEFAULT_TIMEOUT,
+        status: 'yellow',
+      })();
+    }
```

Comment:
Afair this was required to make the test stable. Is that no longer the case? If it's not needed for serverless, do we need it for stateful?

Reply:
I think it's still needed, specially when multiple jobs are being created, but afaik the check is not available in serverless.